src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (a) => {
+    return (b) => {
+      count++;
+
+      if (count > 3 && count % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return a + b;
+      }
```

Comment:
```suggestion
      }
      
      return a + b;
```

Or even you can use ternary operator
`return (count > 3 && count % 2 === 0) ? 'Bzzz... Error!' : a +b`

Reply:
Спасибо, исправила!